src/App.js

```diff
@@ -1,10 +1,19 @@
 import "./App.css";
 import {LandingPage} from "./pages/LandingPage/LandingPage";
+import {Route, Routes} from "react-router-dom";
+import {Login} from "../src/pages/Login/Login";

```

Comment:
Please use a formatter to format your code. Maybe add active Prettier extension on vscode and set **formatOnSave** to true. A bit of googling would help.